app/models/backend/ecs/v2/service_stack.rb

```diff
@@ -1,11 +1,13 @@
 module Backend::Ecs::V2
   class ServiceStack < CloudFormation::Stack
     class Builder < CloudFormation::Builder
+      DEPLOYMENT_CONFIGURATION = { "MaximumPercent"=>150, "MinimumHealthyPercent"=>100 }
```

Comment:
Do you think it will be better to set this to a configurable value, so people can decide if they want lower rolling rate? Perhaps add a variable to the model, but default to 200?